vector/src/main/java/im/vector/app/features/crypto/verification/epoxy/BottomSheetVerificationBigImageItem.kt

```diff
@@ -35,10 +35,23 @@ abstract class BottomSheetVerificationBigImageItem : VectorEpoxyModel<BottomShee
 
     override fun bind(holder: Holder) {
         super.bind(holder)
-        holder.image.render(roomEncryptionTrustLevel)
+        when (roomEncryptionTrustLevel) {
```

Comment:
I would prefer to keep the old code and add a optional parameter in the `ShieldImageView` fun `render(..., noBorder: Boolean = false)` to use alternative image in the case the parameter is true.
Can you do that please?

Reply:
done